website/static/js/home.js

```diff
@@ -0,0 +1,8 @@
+var $ = require('jquery');
```

Comment:
Also, should this file go in the `js/pages` directory since it's a webpack entry point?